### PR TITLE
added tests for accordion

### PR DESCRIPTION
**Pull request by martypowell**
Added tests to match https://github.com/baltimorecounty/BCPL-assets/wiki/Accordions-Test-Plan

### CODE REVIEW

File: cypress/integration/accordions.spec.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    beforeEach(() => {
+        cy.get(constants.selectors.accordionContainer).as('accordions');
+        cy
+            .get('@accordions')
+            .find(constants.selectors.accordionPanel)
+            .first()
+            .as('firstAccordion');
+    });
+
+    it('should at least one accordion on the page', () => {
```
Comment (jsmith176) [inline cypress/integration/accordions.spec.js]: Maybe use 'should _be_ at least one...'

File: cypress/integration/accordions.spec.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            .should('exist');
+    });
+
+    it('should show a down arrow icon when the accordion is expanded', () => {
+        getFirstAccordionToggleBtn()
+            .find('i')
+            .should('have.class', constants.cssClasses.accordionExpandedArrow);
+    });
+
+    it("should collapse an expanded accordion has it's header selected", () => {
```
Comment (jsmith176) [inline cypress/integration/accordions.spec.js]: Use single quotes

Comment (jsmith176) [reply cypress/integration/accordions.spec.js]: Try rephrasing this. Maybe 'should collapse an expanded accordion if it's header is selected'

File: cypress/integration/accordions.spec.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        toggleFirstAccordion()
+            .closest(constants.selectors.accordionPanel)
+            .find(constants.selectors.accordionExpanded)
+            .should('not.exist');
+    });
+
+    it('should show a down arrow icon when the accordion is expaned', () => {
```
Comment (jsmith176) [inline cypress/integration/accordions.spec.js]: You have this test twice. Did you mean to change this to check for a right arrow when the accordion is collapsed?

File: cypress/integration/accordions.spec.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+        getFirstAccordionToggleBtn()
+            .find('i')
+            .should('have.class', constants.cssClasses.accordionCollapsedArrow);
+    });
+
+    it("should collapse an expanded accordion has it's header selected", () => {
```
Comment (jsmith176) [inline cypress/integration/accordions.spec.js]: Use single quotes. Also, I'm not sure I understand this test. You are checking if '@firstAccordion' exists (which is already done above lines 45-47) but the description says it should be checking if the accordion has collapsed.

File: cypress/integration/accordions.spec.js (hunk 1 above)
Comment (martypowell) [reply cypress/integration/accordions.spec.js]: done

**Comment by martypowell** [comment]
@jsmith176 vscode is changing the single quotes to double quotes on a format (i can show you if you want), so in that case lets not worry about that now, formatting  than that issue. all other items addressed

File: cypress/integration/accordions.spec.js [hunk 5]
```diff
@@ -66,13 +66,13 @@ describe('Accordion Component', () => {
             .should('not.exist');
     });
 
-    it('should show a down arrow icon when the accordion is expaned', () => {
+    it('should show a right arrow icon when the accordion is expaned', () => {
```
Comment (jsmith176) [inline cypress/integration/accordions.spec.js]: Should this be 'should show a right arrow icon when the accordion is _**collapsed**_'

File: cypress/integration/accordions.spec.js (hunk 2 above)
Comment (ascire) [reply cypress/integration/accordions.spec.js]: Also since we're correcting grammer, use "its" instead of "it's".

**Comment by martypowell** [comment]
@jsmith176 @ascire ready for another look.

File: cypress/integration/accordions.spec.js [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+            .should('have.attr', constants.attrs.expanded, 'true');
+    });
+
+    it('should scroll to the accessability section', () => {
+        cy.wait(1000); // give the test some time to get down the page
+
+        cy.document().then(doc => {
+            const docPosition = doc.documentElement.scrollTop;
+
+            expect(docPosition).to.be.above(500); // This is just arbitray since I wasn't able to get the anchors offset correctly
```
Comment (ascire) [inline cypress/integration/accordions.spec.js]: I'm not picking on you here but pointing out the absurdity of this object structure.  What purpose does `.to.be` serve here other than as a natural language aid?

Comment (martypowell) [reply cypress/integration/accordions.spec.js]: @ascire i do find it a bit verbose as well, but im just playing by the rules.

https://docs.cypress.io/guides/references/assertions.html#Sinon-Chai

**Pull request merged**